multichain-testing/scripts/create-osmosis-pool.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+set -euo pipefail
+shopt -s expand_aliases
+
+alias agoric-exec="kubectl exec -i agoriclocal-genesis-0 -c validator -- agd"
+alias osmosis-exec="kubectl exec -i osmosislocal-genesis-0 -c validator -- osmosisd"
+alias hermes-exec="kubectl exec -i hermes-agoric-osmosis-0 -c relayer -- hermes"
+
+AGORIC_WALLET="test1"
+AGORIC_ADDRESS=$(agoric-exec keys show ${AGORIC_WALLET} -a)
+OSMOSIS_WALLET="test1"
+OSMOSIS_ADDRESS=$(osmosis-exec keys show ${OSMOSIS_WALLET} -a)
+
+AGORIC_OSMOSIS_CHANNEL=$(hermes-exec --json query channels --show-counterparty --chain agoriclocal \
+  | jq -r '[.][] | select(.result) | .result[] | select(.chain_id_b == "osmosislocal") | .channel_a')
+AGORIC_OSMOSIS_PORT="transfer"
+AGORIC_TOKEN_DENOM="ubld"
+AGORIC_TOKEN_AMOUNT="250000000000"
+
+POOL_CONFIG_FILE="pool-config.json"
+POOL_CONFIG_DEST="/opt/pool-config.json"
+
+POOL_ASSET_1_DENOM="uosmo"
+POOL_ASSET_1_AMOUNT="250000"
+POOL_ASSET_1_WEIGHT="1"
+POOL_ASSET_2_DENOM="ibc/E7827844CB818EE9C4DB2C159F1543FF62B26213B44CE8029D5CEFE52F0EE596"
```

Comment:
Sometimes we get `channel-1`, others `channel-0` for osmosis<>agoric, so I think we want something more dynamic? 

Trying to think of the best way. I think there's a query like `denom-trace` that does the opposite.  Another option is to query the osmosis account balances after the agoric transfer - the hash should be there.